thorlcr/activities/loop/thloopslave.cpp

```diff
@@ -884,9 +884,15 @@ class CConditionalActivity : public CSlaveActivity
         return ret.getClear();
     }
     virtual bool isGrouped() const override { return container.queryGrouped(); }
-    virtual void getMetaInfo(ThorDataLinkMetaInfo &info) override
+    virtual void getMetaInfo(ThorDataLinkMetaInfo &info) const override
     {
         initMetaInfo(info);
+        if (hasStarted())
```

Comment:
shouldn't this be an internal error?  getMetaInfo should never be called if it hasn't been started